pkg/api/types.go

```diff
@@ -2049,6 +2063,23 @@ type ServiceProxyOptions struct {
 	Path string
 }
 
+// OwnerReference contains enough information to let you identify an owning
+// object. Currently, an owning object must be in the same namespace, so there
+// is no namespace field.
+type OwnerReference struct {
+	// API version of the referent.
+	APIVersion string `json:"apiVersion,omitempty"`
```

Comment:
Version is useful, because a kind may not exist in all versions of a group, though we don't such cases yet.

I think it makes sense to break APIVersion into two fields.